src/main/java/com/google/openthread/brski/Voucher.java

```diff
@@ -183,20 +183,25 @@ public int getValue() {
 
   public static final String PINNED_DOMAIN_CERT = "pinned-domain-cert";
 
+  public static final String PINNED_SHA256_DOMAIN_SPKI = "pinned-sha256-of-subject-public-key-info";
+
   public static final String PINNED_DOMAIN_SPKI = "pinned-domain-subject-public-key-info";
 
   public static final String PRIOR_SIGNED_VOUCHER_REQUEST = "prior-signed-voucher-request";
 
   public static final String PROXIMITY_REGISTRAR_CERT = "proximity-registrar-cert";
 
+  public static final String SHA256_REGISTRAR_SPKI =
+      "mity-registrar-sha256-of-subject-public-key-info";
```

Comment:
One more comment here, the string should be "proximity-registrar-sha256-of-subject-public-key-info". In the IETF draft, the table printing caused the first 5 chars to drop off.

Reply:
@wgtdkp , are you planning to fix this in this PR? Or should we merge this PR first?

Reply:
Sorry that I missed this comment, I fixed the typo here. Please take a look later. Thanks!